Cargo.toml

```diff
@@ -20,12 +20,13 @@ persistent_store = { path = "libraries/persistent_store" }
 byteorder = { version = "1", default-features = false }
 arrayref = "0.3.6"
 subtle = { version = "2.2", default-features = false, features = ["nightly"] }
-# This import explicitly locks the version.
-serde_json = { version = "=1.0.69", default-features = false, features = ["alloc"] }
 embedded-time = "0.12.1"
 arbitrary = { version = "0.4.7", features = ["derive"], optional = true }
 rand = { version = "0.8.4", optional = true }
 ed25519-compact = { version = "1", default-features = false, optional = true }
+# We import to explicitly lock the version.
+# Remove these dependencies once CTAP is a library.
+serde_json = { version = "=1.0.69", default-features = false, features = ["alloc"] }
```

Comment:
It's another case were I had to lock the version because of compilation issues. I can probably move it to dev-dependencies, let me try.

Reply:
But I don't understand where do we use `serde_json`?

Reply:
Yes, I've seen that PR and would have mentioned that `serde_json` doesn't need to be added there because we shouldn't depend on it.

I just did the following without issues:
```
./reset.sh
# Remove serde_json from Cargo.toml
./setup.sh
./run_desktop_tests.sh
```

So I'm really curious why we need `serde_json` in the ctap2 crate.

Reply:
In both stable and develop, `serde` is part of the initial commit for `libraries/crypto`. Copied from stable:

```
serde = { version = "1.0", optional = true, features = ["derive"] }
serde_json = { version = "1.0", optional = true }
```

I never checked if we actually need these dependencies, but I removed all of them and it still seems to work. Interesting.

Reply:
The crypto library uses them for tests (wycheproof). But ctap2 doesn't.

Reply:
Oh, we responded at the same time it seems. I had the same conclusion, so let's try to remove serde!